app/src/main/java/com/kickstarter/models/User.kt

```diff
@@ -379,6 +387,7 @@ class User private constructor(
                 unreadMessagesCount() == obj.unreadMessagesCount() &&
                 unseenActivityCount() == obj.unseenActivityCount() &&
                 weeklyNewsletter() == obj.weeklyNewsletter()
+            needsPassword() == obj.needsPassword()
```

Comment:
missing the "&&" operator, that new line is not gonna be executed

Reply:
Fixed